test/regression-tests/issue-5.js

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+assert(Function.length == 1);
+assert(Function.prototype.length == 0);
```

Comment:
Shouldn't these asserts be _after_ the `Function...` statements?

Reply:
Actually I do not think it matters, the original testcase anyways was testing something not related to the above asserts in JerryScript. It just caused an assertion failure, however the above asserts are testing if `Function.length` and `Function.prototype.length` have the right values.

Reply:
Hm, I'm not completely convinced. I just tried the following:
```
escargot> Function.prototype.length = function() { };
function () { }
escargot> Function.prototype.length
0
```

That is, even _after_ assigning a function to `Function.prototype.length`, when I read it, I get 0. Although quite improbable, if the assert was before the assignment, it could miss a changed `length`.